lua/dap/repl.lua

```diff
@@ -197,7 +214,45 @@ local function print_threads(threads)
 end
 
 
+---- Handle multiline inputs
+--
+-- Returns nil if input is not complete, otherwise returns the full user input
+local function handle_followup_input(curr_text)
+  local session = get_session()
+  if not session then
+    return curr_text
+  end
+  local continue_prompt_fn = session.adapter.followup_func_cb
+  if not continue_prompt_fn then
+    return curr_text
+  end
+  local prompt_state = vim.fn.getbufvar(repl.buf, PROMPT_STATE)
+  if prompt_state == REGULAR_PROMPT and continue_prompt_fn(curr_text) then
+    vim.fn.setbufvar(repl.buf, PROMPT_STATE, FOLLOWUP_PROMPT)
```

Comment:
Could use `vim.b[repl.buf]` instead of `getbufvar` and `setbufvar`. See `:h vim.b`